log10/bigquery.py

```diff
@@ -0,0 +1,67 @@
+from google.cloud import bigquery
+from google.api_core.exceptions import NotFound
+import os
+
+
+# todo: add requirements.txt file
+# todo: add instructions for bigquery integration
+
+def initialize_bigquery():
+
+    # Configure the BigQuery client
+    project_id = os.environ.get("PROJECT_ID")
```

Comment:
Mind if prepend `LOG10_` to these environment variables. Or maybe `LOG10_BQ_` to be sure we can do different kinds of data stores